.editorconfig

```diff
@@ -12,6 +12,3 @@ end_of_line = lf
 [*.md]
 max_line_length = off
 trim_trailing_whitespace = false
-
```

Comment:
would keep this one even if I dont use makefiles

Reply:
updated